app/apps/client/orchestrator.js

```diff
@@ -69,12 +69,13 @@ class AppClientOrchestrator {
 
 	getAppsFromMarketplace = async () => {
 		const appsOverviews = await APIClient.get('apps', { marketplace: 'true' });
-		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType, isEnterpriseOnly }) => ({
+		return appsOverviews.map(({ latest, price, pricingPlans, purchaseType, isEnterpriseOnly, modifiedAt }) => ({
```

Comment:
I suggest creating a metho mapFilter that receive a complex object and return this result, in this way make it is easy to understand this block.